documentation/DEVELOPMENT.md

```diff
@@ -41,6 +41,12 @@ You can run RSpec and e2e tests with
 script/test.sh
 ```
 
+We also support Javascript unit testing, with Enzyme for snapshots. 
+
+```
+    docker-compose exec web yarn test 
```

Comment:
The `script/test.sh` runs both the Rails unit tests and the e2e integration tests. It might be nice to _also_ run this in that test script so it's full gamut, and then detail how they can run each individual suite separately (like this one).